tests/foreman/longrun/test_oscap.py

```diff
@@ -45,6 +45,7 @@ def setUpClass(cls):
 
         """
         super(OpenScapTestCase, cls).setUpClass()
+        cls.old_task_timeout = entity_mixins.TASK_TIMEOUT
```

Comment:
similar explanation as above.  Currently this is used in 1 place only, but in future it may be used more.